packages/@uppy/core/src/Uppy.js

```diff
@@ -32,12 +32,6 @@ if (typeof AggregateError === 'undefined') {
     }
   }
 }
-class AggregateRestrictionError extends AggregateError {
-  constructor (...args) {
-    super(...args)
-    this.isRestriction = true
-  }
-}
```

Comment:
Because it’s unused, how do you suggest we use it? We used to show an error info message with all the missing meta fields errors — but not anymore. I thought to log it maybe.

Reply:
I think it can be quite useful for debugging, and in any case it's really cheap to keep it.

Reply:
The linter was complaining that it was unused, do you suggest we log with it?

Reply:
I suggest to revert https://github.com/transloadit/uppy/pull/3285/files#diff-e522340b6dabc4b8f14b055b742c5c1adca74d15a06f06467956a59b7e106cc2L580-R589 and keep using it 😅